### PR TITLE
[13.0] [MIG] partner_firstname: Migration to 13.0

### PR DESCRIPTION
**Pull request by lmignon**
an alternative to #784 where I moved all the context and onchange hacks around the field `name` by taking advantage of the new possibility to declare a computed field with `readonly=False`

### CODE REVIEW

**Comment by lmignon** [comment]
cc @simahawk @Laurent-Corron

**Review by yucer — APPROVED** [review]
Nice! It works for me.

File: partner_firstname/tests/test_user_form.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            # Changes lastname, which triggers onchanges
+            user_form.lastname = lastname
+
+        self.assertEqual(user_form.lastname, lastname)
+        self.assertEqual(user_form.firstname, firstname)
+        self.assertEqual(user_form.name, " ".join((firstname, lastname)))
+
+    def setUp(self):
+        super(UserOnchangeCase, self).setUp()
+        self.user = self.env["res.users"].new()
```
Comment (simahawk) [inline partner_firstname/tests/test_user_form.py]: is this used?

**Review by simahawk — APPROVED** [review]
only code review, LG.

File: partner_firstname/tests/test_user_form.py (hunk 1 above)
Comment (lmignon) [reply partner_firstname/tests/test_user_form.py]: Good catch @simahawk! Code removed.

**Comment by pedrobaeza** [comment]
@NL66278 please use ocabot for merging PRs, as it handles in the same merge operation the updated translations, README and the rest of administrative stuff. See https://odoo-community.org/blog/the-oca-blog-1/post/the-oca-github-bot-can-now-merge-pull-requests-92 for more details.

**Pull request merged**